python/test-requirements.txt

```diff
@@ -5,10 +5,11 @@ py>=1.4.31
 randomize>=0.13
 tox<4.0.0
 
+# Build package
```

Comment:
**explain:** Moved it here, from being a comment in the `Makefile`, so that it's installed as other requirements are